features/step_definitions/networking.rb

```diff
@@ -886,4 +886,23 @@
   @result = host.exec_admin("ifconfig #{cb.interface}")
   cb[cb_ipaddr]=@result[:response].match(/\d{1,3}\.\d{1,3}.\d{1,3}.\d{1,3}/)[0]
   logger.info "The Internal IP of node is stored in the #{cb_ipaddr} clipboard."
-end
\ No newline at end of file
+end
+
+Given /^I store "([^"]*)" node's corresponding default networkType pod name in the#{OPT_SYM} clipboard$/ do |node_name,cb_pod_name|
+  ensure_admin_tagged
+  node_name ||= node.name
+  _admin = admin
+  @result = _admin.cli_exec(:get, resource: "network.operator", output: "jsonpath={.items[*].spec.defaultNetwork.type}")
+  if @result[:response] == "OpenShiftSDN"
+     app="app=sdn"
+     project_name="openshift-sdn"
+  else
+     app="app=ovnkube-node"
+     project_name="openshift-ovn-kubernetes"
+  end   
+  raise "Unable to find corresponding networkType pod name" unless @result[:success]
+  cb[cb_pod_name] = BushSlicer::Pod.get_labeled(app, project: project(project_name, switch: false), user: admin) { |pod, hash|
```

Comment:
@anuragthehatter, we need to assign a default value for `cb_pod_name`, otherwise it will be `nil` and go to the default which might have some values there already.

Reply:
Done, thanks! I usually do that but missed here